packages/lesswrong/server/recombee/client.ts

```diff
@@ -495,11 +499,13 @@ const recombeeApi = {
     const { curatedPostIds, stickiedPostIds, excludedPostFilter } = await helpers.getOnsitePostInfo(lwAlgoSettings, context);
 
     const curatedPostReadStatuses = await helpers.getCuratedPostsReadStatuses(lwAlgoSettings, curatedPostIds, userId, context);
+    const manuallyStickiedPostReadStatuses = await helpers.getManuallyStickiedPostsReadStatuses(stickiedPostIds, userId, context);
```

Comment:
This would filter out all the already-read stickied posts, not just the manually stickied ones (absent the change suggested in https://github.com/ForumMagnum/ForumMagnum/pull/9404/files#r1639108987)